### PR TITLE
Release v0.5.0

### PR DESCRIPTION
**Pull request by brunoocasali**
Pre-release PR:

- Updates multiple versions all over the project to `v0.5.0`
- Add changes to the CHANGELOG

### CODE REVIEW

**Review by curquiza — APPROVED** [review]
👍

**Comment by brunoocasali** [comment]
bors merge

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,33 @@
+# 0.5.0
+This version makes this package compatible with MeiliSearch v0.25.0 and newer
+## Changes
+- Add method to responds with raw information from API `client.getRawIndex` (#124) @brunoocasali
+- Run a `pub upgrade` in dependencies (#128) @brunoocasali
+- Add support to keys (according to v0.25.0 spec) [more](https://docs.meilisearch.com/reference/api/keys.html#keys) (#121) @brunoocasali
+  - Create a new resource class `Key` to represent keys
+  - Create methods to support keys:
+    - `client.getKeys`
```
Comment (alallema) [inline CHANGELOG.md:9]: I'm not totally sure but I think `getKeys` method should be in breaking because the return of the method has changed.

**Review by alallema — COMMENTED** [review]
Just a question otherwise everything look good to me! ✨

File: CHANGELOG.md (hunk 1 above)
Comment (alallema) [reply CHANGELOG.md:9]: You are both way to fast for me 😄

Comment (brunoocasali) [reply CHANGELOG.md:9]: Thanks for raising this @alallema!

But if you scroll down a bit, you'll see the `client.getKeys` in the breaking section too :)

Comment (brunoocasali) [reply CHANGELOG.md:9]: image

**Pull request merged**